examples/ui/font_atlas_debug.rs

```diff
@@ -63,9 +63,8 @@ fn atlas_render_system(
 
 fn text_update_system(mut state: ResMut<State>, time: Res<Time>, mut query: Query<&mut Text>) {
     for mut text in query.iter_mut() {
-        state.timer.tick(time.delta_seconds);
         let c = rand::random::<u8>() as char;
-        if !text.value.contains(c) && state.timer.finished {
+        if !text.value.contains(c) && state.timer.tick(time.delta_seconds).finished {
```

Comment:
this won't actually behave the same because && expressions will short-circuit when the first criteria isn't met. we want to tick every time, whether or not text.value contains `c`

Reply:
Oh right, I’m gonna keep this the old way, I didn’t like that the tick was kinda hidden in this case anyway. Good catch